examples/mnist/Instructions.md

```diff
@@ -0,0 +1,25 @@
+<p align="center"><b>How to test example MNIST train/test programs</b><p align="center">
```

Comment:
I noticed the word `test` is used twice in the above sentece I wrote.
Maybe `How to ready MNIST train/test example programs.` is a bit better?